content/General Support/decentralized_autonomous_organization.en.md

```diff
@@ -0,0 +1,26 @@
+---
+title: What is Decentralized Autonomous Organization (DAO)
+date: "2020-03-12"
+draft: false
+---
+
+A decentralized autonomous organization (DAO) is an organization represented by rules encoded as a computer program that is transparent, controlled by shareholders and not influenced by a central government. NavCoin's DAO intend to achieve this with three major features:
+
+
+	1) Community Funding System
+	The community funding system aims at creating a decentralized funding system where anyone in the world can submit proposals. The proposals will then be voted on by the network. Once accepted, the requested funds will be locked up for the proposal. Payment requests can then be submitted to release the funds in small portions or all at once. This funding system is unique in many ways:
+		a. It is decentralized: 
+		There is no "dev team" or anyone controlling the funds. The only way to get access to the funds is through network consensus. There is also no limitation to who can vote. Anyone owning NAV and staking has the ability to vote. When a block is staked, the staker will cast the vote. There is no minimum staking amount requirement or masternodes for NavCoin. You stake, you vote.
+		b. It is multi-consensus: 
```

Comment:
What about `Two rounds of voting`?

Reply:
It was how we have been saying but it might not always be accurate if there are many payment requests. What do you think?

Reply:
After some thinking i think two rounds of voting is good. Not too technical but accurate at the same time.